recipe/migrations/pypy38.yaml

```diff
@@ -0,0 +1,53 @@
+migrator_ts: 1647123563
+__migrator:
+    migration_number: 1
+    primary_key: python
+    ordering:
+        python:
+            - 3.6.* *_cpython
+            - 3.7.* *_cpython
+            - 3.8.* *_cpython
+            - 3.9.* *_cpython
+            - 3.10.* *_cpython
+            - 3.6.* *_73_pypy
+            - 3.7.* *_73_pypy
+            - 3.8.* *_73_pypy
+            - 3.9.* *_73_pypy
+    paused: False
+    longterm: True
+    use_local: False
+    check_solvable: True
+    exclude_pinned_pkgs: False
+    pr_limit: 1
+    bump_number: 1
+    wait_for_migrators:
+      - python310
+    commit_message: "Rebuild for PyPy3.8 and PyPy3.9"
+    exclude:
+      # this shouldn't attempt to modify the python feedstocks
+      - python
+      - pypy3.6
+      - pypy-meta
+
+python:
+  - 3.7.* *_cpython    # [not (osx and arm64)]
+  - 3.8.* *_cpython
+  - 3.9.* *_cpython
+  - 3.10.* *_cpython
+  - 3.8.* *_73_pypy    # [not (osx and arm64)]
+  - 3.9.* *_73_pypy    # [not (osx and arm64)]
+numpy:
+  # part of a zip_keys: python, python_impl, numpy
+  - 1.19               # [not (osx and arm64)]
+  - 1.19
+  - 1.19
+  - 1.21
+  - 1.22               # [not (osx and arm64)]
+  - 1.22               # [not (osx and arm64)]
```

Comment:
I'd be willing to backport the pypy-migrator to numpy 1.19, which supports python 3.8/3.9

Reply:
No need. We don't support older software on newer platforms anyway.

Reply:
We did do it for win-pypy (though more or less accidentally at the time as the migrator had set the version too low, and I just did the backport), and it [used to](https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/2008#discussion_r729339685) be done more for cpython releases as well (though cpython is different here as numpy itself had e.g. no 3.10 support before 1.21, but pypy3.8 should - in principle - work without issues with 1.19).

I'm probably going to attempt it anyway on the numpy repo after the migrator runs through - if it works, I presume the migrator can anyway be changed while it's running...?

IOW, fine to merge as is from my POV.